### ISSUE TITLE
Pick behavior not working

### ISSUE THREAD

**Issue by MacLotsen** [Josef-Friedrich/luakeys#22]
When having the following parser defined:
```lua
local parse_commit_opts = luakeys.define({
    rev_spec = { data_type = 'string', pick = 'string' },
    files = { data_type = 'list' },
    cwd = { data_type = 'string' },
    flags = {
        sub_keys = {
            merges = { data_type='boolean', exclusive_group='merges' },
            ['no-merges'] = { data_type='boolean', exclusive_group='merges' }
        }
    }
})
```
And execute with `parse_commit_opts('0.1.0...0.0.1,flags={no-merges}')` it works as expected.
However, when passing a single rev_spec, like `parse_commit_opts('0.0.1,flags={no-merges}')`, `rev_spec` doesn't get picked and doesn't end up in the `unknown` table as well.

I've solved it like [this](https://github.com/Xerdi/gitinfo-lua/commit/23fc77d66a5cb7963422bed7c92b5e17a7ace651) for now.

**Comment by Josef-Friedrich** [comment]
Thank you for reporting this issue! 

Does `pick = 'any'` work?

**Comment by MacLotsen** [comment]
> Thank you for reporting this issue!

You're more than welcome, thanks for making this package!

> Does `pick = 'any'` work?

Unfortunately not :(

**Comment by MacLotsen** [comment]
When I use `v0.0.1` instead of `0.0.1`, then it works.

**Comment by MacLotsen** [comment]
Found a solution! Though, might introduce some bad indentation formatting. All Lua tests still run (see #23)

**Comment by MacLotsen** [comment]
Solved in #23.
